src/Message.cpp

```diff
@@ -43,4 +43,7 @@ namespace dpp {
 	std::string Message::add_mention(const std::string& emoji) {
 		return Api::put("/channels/" + channel_id + "/messages/" + id + "/reactions/" + emoji + "/@me").dump();
 	}
-}
\ No newline at end of file
+	bool Message::startsWith(const std::string& s) {
+		return content.find(s) == 0;
+	}
```

Comment:
This feature has already been added in recent commits at the `main` branch

Reply:
I know i just do that to be sure that everything is correct